.eslintrc.js

```diff
@@ -5,9 +5,7 @@ module.exports = {
     jest: true,
   },
   parser: 'babel-eslint',
-  extends: [
-    'airbnb',
-  ],
+  extends: ['airbnb', 'prettier', 'prettier/prettier'],
```

Comment:
These both are needed since is to add the extension 'prettier'. 'prettier/prettier'  is added to configure some rules as described here